scripts/sexbound/plugins/pregnant/pregnant.lua

```diff
@@ -834,6 +923,8 @@ end
 
 --- Validates the loaded config and sets missing config options to be default values.
 function Sexbound.Actor.Pregnant:validateConfig()
+    self:validateDripTimer(self._config.dripTimer)
```

Comment:
Having an option to configure the drip decay timer is a good idea - however, there is no actual config value added that represents this.

Reply:
Yes, the thing is *I* can set it. But a user should never have to try and edit the code themselves. They already implode their game enough trying to edit the config.

I'm sure a setting for this specific part is not necessary, but it also wouldn't surprise me when people start asking rather sooner than later if they can adjust the values, so I brought it up as a pointer.